Backend/src/main/java/idealab/api/operations/DropboxOperations.java

```diff
@@ -99,10 +110,63 @@ public void deleteDropboxFile(PrintJob printJob){
     // 1. Delete existing dropbox file using deleteDropboxFile() method
     // Note:  if the file path does not start with a "/" then there is either an error or it was deleted already.
     if(printJob.getDropboxPath().startsWith("/")){
-      deleteDropboxFile(printJob);
+      deleteDropboxFile(printJob.getDropboxPath());
     }
     // 2. Create new sharable data link using uploadDropboxFile() method
     LocalDateTime currentTime = LocalDateTime.now();
     return uploadDropboxFile(currentTime.toLocalTime().toNanoOfDay(), file);
   }
+
+  public PrintJobResponse updateDropboxPath(DropBoxFilePathRequest request) {
+    request.validate();
+
+    PrintJob printJob = printJobRepo.findPrintJobById(request.getPrintJobId());
+
+    if(printJob == null)
+      throw new IdeaLabApiException(PRINT_JOB_CANT_FIND_BY_ID);
+
+    LocalDateTime currentTime = LocalDateTime.now();
+    String newPath = "/" + currentTime.toLocalTime().toNanoOfDay() + request.getNewPath();
+    String oldPath = printJob.getDropboxPath();
+
+    try {
+      //Copy file to new location in dropbox, delete old file from dropbox, update database
+      RelocationResult result = client.files().copyV2(oldPath, newPath);
```

Comment:
Instead of copy do you think we should use the move api? 

https://www.dropbox.com/developers/documentation/http/documentation#files-move

Reply:
creating issue so we dont lose this.

Reply:
made issue #203